Source/WebCore/platform/mediarecorder/MediaRecorderPrivateEncoder.cpp

```diff
@@ -411,20 +414,19 @@ void MediaRecorderPrivateEncoder::audioSamplesAvailable(const MediaTime& time, s
 
 void MediaRecorderPrivateEncoder::appendVideoFrame(VideoFrame& frame)
 {
-    if (m_isStopped)
+    if (m_isStopped || m_isPaused)
```

Comment:
We somehow loose a few frames when resuming.
In theory, we should probably check for m_isPaused in the lambda below.
And we could capture `now()` to set m_resumeWallTime if needed.